.github/workflows/binaries.yml

```diff
@@ -11,16 +11,16 @@ jobs:
       matrix:
         include:
           - os: macos-13
-            name: mac-intel
+            name: x86_64-apple-darwin
             installable: .#
           - os: macos-14
-            name: mac-arm
+            name: aarch64-apple-darwin
             installable: .#
           - os: ubuntu-22.04
```

Comment:
I thought we got rid of that one, maybe that was only on the toolchains branch

Reply:
Do you want me to remove this target or we leave it here?

Reply:
can you do it in a separate PR so we don't forget?

Reply:
Sure, no problem :+1: 
Done in #10683